packages/paste-design-tokens/tokens/global/palette-color.yml

```diff
@@ -38,6 +38,186 @@ props:
   color-gray-100:
     value: "{!palette-gray-100}"
     comment: "Gray Color 10"
+  color_red_1:
```

Comment:
OK so, aliases should not be added to the theme. When you create or add tokens, you will automatically export them as a public API on the design token package, and then automatically add them to the React theme context in the Theme Package. Effectively adding them as a public API to the theme package too. Then, depending on the type of token you create, you might automatically add it to Box style props, and it's typings. (The color bucket is the only bucket of tokens that is not assigned to a style props right now, so this specific example isn't exposed automatically at the Box theme).

What you are doing in this file is actually creating Design Tokens. What you want to do instead is make sure to add any missing _aliases_ to the color palette aliases. https://github.com/twilio-labs/paste/blob/main/packages/paste-design-tokens/tokens/aliases/color-palette.yml

These are then available in https://unpkg.com/browse/@twilio-paste/design-tokens@7.0.0/dist/tokens.raw.json which is exported by the design tokens package.

You could probably then add that to the gatsby graphql if you like, but importing that json file from design tokens is probably just enough.

`import {aliases} from '@twilio-paste/design-tokens/dist/tokens.raw.json'`
`import {aliases} from '@twilio-paste/design-tokens//dist/themes/dark/tokens.raw.json'`

Reply:
Ohhhhhhhhhhhhhhhhhhhhhh ok. These files are all very similar so i guessed where to add them 😬